generate/generate.go

```diff
@@ -58,6 +58,20 @@ var detailsRequireZeroIndex = map[string]bool{
 	"updateAccount":    true,
 }
 
+// requiresPost is a prefilled set of API names that require POST
+// for security or size purposes
+var requiresPostMethod = map[string]bool{
+	"login":                            true,
+	"deployVirtualMachine":             true,
+	"updateVirtualMachine":             true,
+	"createUser":                       true,
+	"updateUser":                       true,
+	"addVpnUser":                       true,
+	"registerUserData":                 true,
+	"setupUserTwoFactorAuthentication": true,
+	"validateUserTwoFactorAuthenticationCode": true,
```

Comment:
The code for these 3 commands is not being generated since they are not defined in `generate/layout.go`.

Reply:
Correct.